src/gui/templates/contact.html

```diff
@@ -1,21 +1,44 @@
 {% extends "base.html" %}
 {% load static %}
 {% block content %}
-      <h2>Kontakt</h2>
+<h2>Kontakt</h2>
 {% if sent %}
-      <div class="content-item centered">
-        <p>Nachricht erfolgreich versandt.</p>
-      </div>
+  <div class="content-item centered">
+    <p>Nachricht erfolgreich versandt.</p>
+  </div>
 {% endif %}
-      <div class="content-item centered">
-        <p>Nimm Kontakt zum Team auf. Die Rückmeldung erhältst du per E-Mail.</p>
-        <p>Alternativ kannst du auch eine E-Mail an <a href="mailto:bunterball@insafehands.de">bunterball@insafehands.de</a> schreiben.
-      </div>
-      <div class="centered">
-        <form method="post" action="/gui/send_contact/">
-          {% csrf_token %}
-          <textarea style="margin:auto; width: 95%; height: 40vh; resize:none; margin-bottom: 0.3em;" name="message"></textarea>
-          <input type="submit" class="ig-button" value="Senden">
-        </form>
-      </div>
+<div class="content-item centered">
+  <p>Nimm Kontakt zum Team auf. Die Rückmeldung erhältst du per E-Mail.</p>
+  <p>Alternativ kannst du auch eine E-Mail an <a href="mailto:bunterball@insafehands.de">bunterball@insafehands.de</a> schreiben.
+</div>
+<div class="centered">
+  <form method="post" action="/gui/send_contact/">
+    {% csrf_token %}
+    <textarea style="margin:auto; width: 95%; height: 40vh; resize:none; margin-bottom: 0.3em;" name="message"></textarea>
+    <input type="submit" class="ig-button" value="Senden">
+  </form>
+</div>
+
+<h2>Projektleitung</h2>
+<ul>
+{% for user in leader %}
+    <li>
+      {{ user.avatar }}
+      {{ user.author.first_name }} {{ user.author.last_name }}
+      {{ user.telephone }}
+    </li>
+    {% endfor %}
+</ul>
+
+<h2>Team</h2>
+<ul>
+  {% for user in team %}
+    <li>
+      {{ user.avatar }}
```

Comment:
Does it make sense to do a little more styling here? In this case we have everything in one line, right?